Algorithms/Search/BFS.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections.Generic;
+
+namespace BFS
+{
+    /// <summary>
+    /// Bread First Search implementation.
+    /// </summary>
+    public class BreadthFirst<T>
+    {
+        // Returns true if item exists.
+
+        public bool Find(IGraph<T> graph, T vertex)
+        {
+            return bfs(graph.ReferenceVertex, new HashSet<T>(), vertex);
+        }
+
+        // BFS implementation.
+
+        private bool bfs(IGraphVertex<T> referenceVertex,
+            HashSet<T> visited, T searchVertex)
+        {
+            var bfsQueue = new Queue<IGraphVertex<T>>();
+
+            bfsQueue.Enqueue(referenceVertex);
+            visited.Add(referenceVertex.Key);
+
+            while (bfsQueue.Count > 0)
+            {
+                var current = bfsQueue.Dequeue();
+
+                if (current.Key.Equals(searchVertex))
+                	return true;

```

Comment:
Codacy found an issue: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6325512)